### PR TITLE
Refactored WP_Debug_Data::debug_data() Part 3

### PR DESCRIPTION
**Pull request by kebbet**
https://core.trac.wordpress.org/ticket/61648

Follow up to #7106

### CODE REVIEW

**Review by apermo — APPROVED** [review]
Looks good to me.

File: src/wp-admin/includes/class-wp-debug-data.php [hunk 1]
```diff
@@ -29,13 +29,12 @@ public static function check_for_updates() {
 	 * @since 5.5.0 Added pretty permalinks support information.
 	 *
 	 * @throws ImagickException
-	 * @global wpdb  $wpdb               WordPress database abstraction object.
```
Comment (dmsnell) [inline src/wp-admin/includes/class-wp-debug-data.php:32]: nice! I know this is just moving it around, but it's nice seeing it leave the bigger general function and become isolated in the DB-related function.

**Review by dmsnell — COMMENTED** [review]
Thanks @kebbet! Looks like another good change on this project.

**Review by dmsnell — APPROVED** [review]

**Comment by dmsnell** [comment]
Merged in [[58964]](https://core.trac.wordpress.org/changeset/58964)
https://github.com/wordpress/wordpress-develop/commit/68805bfe3d05c0789fbe62f6bdb21230c945d295

**Pull request closed without merging**